tests/m2m_through_regress/tests.py

```diff
@@ -47,41 +47,17 @@ def test_retrieve_forward_m2m_items(self):
             ]
         )
 
-    def test_cannot_use_setattr_on_reverse_m2m_with_intermediary_model(self):
-        msg = (
-            "Cannot set values on a ManyToManyField which specifies an "
-            "intermediary model. Use m2m_through_regress.Membership's Manager "
-            "instead."
-        )
-        with self.assertRaisesMessage(AttributeError, msg):
-            self.bob.group_set.set([])
+    def test_set_on_reverse_m2m_with_intermediary_model(self):
+        self.bob.group_set.set([])
```

Comment:
Are these tests checking something that the `m2m_through` tests aren't? It looks odd that the results of these calls aren't verified.

Reply:
Yeah, good point. I was mostly just converting "can't" test into "can" tests. Should I just delete them these?

Reply:
I didn't look closely but I don't think they add anything (but please audit test coverage as I said in my other comment).

Reply:
Ok, I removed these tests.